api/src/main/java/com/messagebird/objects/VoiceStep.java

```diff
@@ -10,6 +13,11 @@
     private String action;
     private VoiceStepOption options;
 
+    private VoiceCallCondition[] conditions;
```

Comment:
I am not sure about conditions object. On dev doc, these fields are not in object definition info but I also saw those are on JSON request. I want to leave a question mark to here :D Let's check every other repository together for Voicestep object

Reply:
Sure, we can. 
Here the GO client ref: `https://github.com/messagebird/go-rest-api/blob/master/voice/callflow.go#L178`